lib/usb_connection.js

```diff
@@ -228,10 +228,13 @@ util.inherits(USB.Scanner, EventEmitter);
 
 USB.Scanner.prototype.start = function() {
   var self = this;
+  if (haveusb) {
```

Comment:
We shouldn't have to need this check because any tests that might use T2 fetching should mock the discovery process.